students/grant_dowell/Lesson_07/html_render.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
+
+"""
+A class-based system for rendering html.
+"""
+
+
+# This is the framework for the base class
+class Element(object):
+
+    tag = "html"
+    tag_attributes = dict()
```

Comment:
this should not be a class variable --- if you are setting it via `__init__` method then it's an instance attribute.